wtxmgr/db.go

```diff
@@ -1215,18 +1215,39 @@ func (it *unminedCreditIterator) reposition(txHash *chainhash.Hash, index uint32
 //   [0:32]   Transaction hash (32 bytes)
 
 func putRawUnminedInput(ns walletdb.ReadWriteBucket, k, v []byte) error {
-	err := ns.NestedReadWriteBucket(bucketUnminedInputs).Put(k, v)
+	spendTxHashes := ns.NestedReadBucket(bucketUnminedInputs).Get(k)
```

Comment:
The `godoc` comment should be updated to reflect that we now store a slice of txid's rather than a single unique txid.

Reply:
Fixed.